database_schemas/cassandra/benchflow.cql

```diff
@@ -23,14 +23,26 @@ CREATE TABLE database_sizes (
   trial_id text,
   dbms text,
   database_name text,
-  database_table text,
   size bigint,
   PRIMARY KEY ((trial_id, experiment_id), dbms)
 );
 
+CREATE TABLE io_data (
+  experiment_id text,
+  trial_id text,
+  container_properties_id uuid,
+  device text,
+  writes bigint,
+  reads bigint,
+  sync bigint,
+  async bigint,
+  total bigint,
+  PRIMARY KEY ((trial_id, experiment_id), container_properties_id)
+);
+
 CREATE TABLE environment_data (
   environment_data_id uuid,
-  container_properties_id uuid,
+  container_id text,
```

Comment:
This should be `container_properties_id` because the only unique id we have is the one we generate. It is possible, although unlikely, that in the same `experiment` and `trial` we have two containers with the same `container_id`.